### PR TITLE
Implemented google recaptcha

### PR DESCRIPTION
**Pull request by efatsi**
Allows gem users to configure a Google Captcha Site Key and Secret Key. With which it uses to generate the smart Google "I am not a robot" thinger.

Upon clicking "I am not a robot", a `g-recaptcha-response` parameter is added to the form post, which you then make an API request with to confirm the humanness.

Take that robots.

### CODE REVIEW

**Comment by mackermedia** [comment]
https://www.youtube.com/watch?v=S_oMD6-6q5Y

File: app/views/pointless_feedback/messages/new.html.erb [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<% if PointlessFeedback.using_captcha? %>
+  <head>
+    <script src='https://www.google.com/recaptcha/api.js'></script>
+  </head>
+<% end %>
```
Comment (mackermedia) [inline app/views/pointless_feedback/messages/new.html.erb]: Will this make the DOM like `<head></head>` -> `<body><head></head></body>` if using typical Rails application layout? Might move to not be in the `<head>` if so?

**Comment by mackermedia** [comment]
Neat-o :+1:

File: app/views/pointless_feedback/messages/new.html.erb (hunk 1 above)
Comment (efatsi) [inline app/views/pointless_feedback/messages/new.html.erb]: Rails magix - this actually ends up in the proper `<head>`.

Comment (ltk) [inline app/views/pointless_feedback/messages/new.html.erb]: woah, really?

**Comment by ltk** [comment]
I'd probs unit test that `Captcha` class, but :+1:

**Comment by efatsi** [comment]
ps. thanks

**Pull request merged**